src/main.c

```diff
@@ -1227,10 +1272,67 @@ static void handle_get_component_line(void *data, const char *line) {
   }
 }
 
+static char* get_get_launch_components_cmd(char* sharedenv) {
```

Comment:
```suggestion
static char* get_launch_components_cmd(char* sharedenv) {
```
Could we rename this method? See get_get_

Reply:
sure, it was because the command is get-launch-components, to follow the same as the similar function for the prepare-instance